app/controllers/v0/attachments_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module V0
-  class AttachmentsController < SMController
+  class AttachmentsController < MyHealth::SMController
```

Comment:
We don't want these controllers to inherit from `MyHealth`, because versions of these controllers have already been copied over to `MyHealth` module. These controllers should be deleted as well.

Basically, the PRs should be:
- Removal of SMController, and the controllers that inherit from it
- Removal of BBController, and the controllers that inherit from it
- Removal of RXController, and the controllers that inherit from it (this one will be complicated because of specs that rely on the controller when they should be using an arbitrary controller to test)